core/peer/peer.go

```diff
@@ -322,6 +322,14 @@ func (p *PeerImpl) GetPeers() (*pb.PeersMessage, error) {
 	return peersMessage, nil
 }
 
+func getPeerAddresses(peersMsg *pb.PeersMessage) []string {
+	var addresses []string
+	for _, v := range peersMsg.GetPeers() {
+		addresses = append(addresses, v.Address)
```

Comment:
Fixed in https://github.com/hyperledger/fabric/pull/2016/commits/91aa90b9f4fbe3452510398f1745b5127367bebb